docs/api/proposed_endpoints.md

```diff
@@ -0,0 +1,228 @@
+[TOC]
+
+# Social Media Posts
+
+## Create a post
+
+```
+POST /api/posts (requires authentication)
+```
+
+**Parameters**
+
+Name          | Data Type   | Description
+--------------|-------------|------------------------------------------
+id            | UUID        | identity of the post
+posted_by     | UUID        | foreign key to `Users` model
+posted_on     | text        | social media ploatform to post
+created_at    | datetime    | time at which user submits the post
+modified_at   | datetime    | time at which the post is modified/updated
+scheduled_time| datetime    | time that the user schedules for his post to be posted
+text          | text        | content of the post
+image         | text        | url of image in the post
+is_approved   | boolean     | whether the post is approved or not
+approved_time | datetime    | time at which the post is approved
+is_posted     | boolean     | whether the post is posted or not
+posted_time   | datetime    | time at which the post will be posted
+
+**Request**
+```json
+{
+    "id": "0f342ac1-ac32-4bd1-3612-efa32bc3d9a0",
+    "posted_by": "01ade2ff-ab21-231f-a12b3c4d5e79",
+    "posted_on": "twitter",
+    "created_at": "2018-08-01T17:30:42Z",
```

Comment:
We don't provide `created_at`. It is handled by `TimeStampedUUIDModel` which our Post model inherits.

Reply:
+1